test/alternative_spec.rb

```diff
@@ -0,0 +1,40 @@
+require_relative "spec_helper.rb"
+require "update-alternatives/model/alternative"
+
+describe UpdateAlternatives::Alternative do
+
+  describe ".all" do
+    subject { UpdateAlternatives::Alternative.all }
```

Comment:
You can also use `described_class.all`

Not really important, but just for you to know.